xbmc/windowing/gbm/OffScreenModeSetting.cpp

```diff
@@ -0,0 +1,42 @@
+#include "OffScreenModeSetting.h"
```

Comment:
@a1rwulf, I imagine the missing copyright notice was not intended. I'll grab it from the header and add it in https://github.com/hudokkow/xbmc/pull/4

Reply:
Oh, of course not.
Thx @hudokkow!!

Reply:
:+1: